src/pages/getting-started/app-guides/index.mdx

```diff
@@ -20,9 +20,17 @@ Apart from the actual code of your application, you need to configure its funcio
 - [Chat Actions](/getting-started/app-guides/chat-actions/)
 - [Chat Webhooks](/app-guides/chat-webhooks/)
 - [In-App Upgrades](/getting-started/app-guides/in-app-upgrades/)
-- [Onboarding & Settings](/getting-started/app-guides/onboarding-and-settings/)
+- [Onboarding](/getting-started/app-guides/onboarding/)
 - 💡[Properties](/getting-started/app-guides/properties/) **(new)**
 
+## Building blocks for HelpDesk
+
+- [App Authorization](/getting-started/app-guides/app-authorization/)
+- [App Monetization](/getting-started/app-guides/app-monetization)
+- [App Webhooks](/getting-started/app-guides/app-webhooks/)
+- [HelpDesk Widgets](/getting-started/app-guides/helpdesk-widgets/)
+- [Onboarding](/getting-started/app-guides/onboarding/)
+
 ## Video overview of Developer Console
```

Comment:
I would rethink the placement of this video - it's not visible when you open the page

Reply:
I have tried a few solutions and, I believe, decided on the best one